.goreleaser.yaml

```diff
@@ -0,0 +1,85 @@
+project_name: apm-lambda-extension
+
+before:
+  hooks:
+    - make check-licenses
+    - make NOTICE.txt
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w
+    binary: "extensions/{{ .ProjectName }}"
+
+archives:
```

Comment:
**question**: is this the spec for this task ? https://goreleaser.com/customization/archive/

Can't you use the `mtime` fields and get rid of the touch workaround ? Or am I missing something ?

Reply:
Oh nice, thank you for that. I was not aware that `mtime` could be used.

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/8975604b5c891a79e4c925b6aadfed9d9c4b53bb